src/extension.ts

```diff
@@ -55,6 +56,12 @@ export function activate(context: vscode.ExtensionContext) {
 
 		await node.createChild();
 	});
+	actionHandler.registerCommand('cosmosDB.deleteAccount', async (node?: IAzureNode) => {
+		if (node) {
```

Comment:
Use the `showNodePicker` method in case the user runs this command from the command palette. Take a look at 'cosmosDB.openInPortal' below. It should be the exact same